pkg/app/sensor/artifacts.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
this needs to be reverted because it requires a newer version of Go and docker-slim still needs to be compatible with Go 1.15